### PR TITLE
[projects] Enable New Workspace button when config is Git-based

### LINKED ISSUE #5591: Allow users to open a new workspace when configuration is Git-based
### Bug description

When configuration is stored in git but is invalid, users importing the project cannot open a new workspace. See also [relevant discussion](https://gitpod.slack.com/archives/C021Q4BL8E8/p1631085405146200) (internal).

### Steps to reproduce

1. Add a project with an invalid configuration (`.gitpod.yml`)
2. In the `/configuration` page you can not run a prebuild due to the configuration error or open a new workspace.

### Expected behavior

Allow users to open a new workspace when configuration is invalid.

### Example repository

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by AlexTugarev**
Enables New Workspace button according to https://github.com/gitpod-io/gitpod/pull/5654#discussion_r710922579

## Related Issue(s)

Fixes #5591

## Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by roboquat** [comment]
@AlexTugarev: Adding the "do-not-merge/release-note-label-needed" label because no release-note block was detected, please follow our [release note process](https://git.k8s.io/community/contributors/guide/release-notes.md) to remove it.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **alextugarev** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by jankeromnes — APPROVED** [review]
Code change looks good to me, many thanks! 👍

Haven't tested yet (CI still in progress)
/hold

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 245aae24a689a450b490abaa8f9eb2cf3bb4c538</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5907#pullrequestreview-765472254" title="Approved">jankeromnes</a>*

Associated issue: *#5591*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by jankeromnes** [comment]
@gtsiolis It says build failed, but it actually succeeded: https://alex-allow-users-to-open-a-new-5591.staging.gitpod-dev.com/workspaces/

**Comment by jankeromnes** [comment]
Huh. For some reason, the `New Workspace` button remains disabled when adding a project with a Git-based config:

Screenshot 2021-09-28 at 17 00 10

**Comment by jankeromnes** [comment]
Aha, new theory: maybe the first build corresponds to an earlier state of the PR, thus didn't actually fix the issue. The latest state of the PR didn't build, so we still can't test it.

/werft run

**Comment by jankeromnes** [comment]
There we go! Works like a charm 😁

| DISABLED | ENABLED | IT WORKS! |
| --- | --- | --- |
| Screenshot 2021-09-28 at 17 50 03 | Screenshot 2021-09-28 at 17 49 35 | Screenshot 2021-09-28 at 17 49 43 |

/lgtm
/hold cancel

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5907#issuecomment-929357726" title="LGTM">jankeromnes</a>*

Associated issue: *#5591*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/dashboard/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/OWNERS)~~ [jankeromnes]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
